### DISCUSSION TITLE
Creating Levels - Placed Staff Members

### DISCUSSION THREAD

**Discussion by dbrown1986** [CorsixTH/CorsixTH#2252] [General Diagnostics]
I'm hoping that this is a good place to post this.

I'm creating my own level, and I've looked through the original campaign and some fan-made levels from CTH, so far I've been able to conclude what each function is written to do, and I have been able to figure out what each function does, such as defining staff_levels for each month and that it can be used to randomly generate staff peeps with certain skills each month or quarterly rotation, etc etc.

However, I want my level to START with three triple-specialized consultants already employed, two for general practice, and one for training, and the rest are hired on as mostly low-level juniors and doctors.

I have seen the main game do this (Simpleton level starts you off with three consultants who are each specialized in ONE specialization), and I know of at least one fan mission that does this. However, when I looked through EASY05.SAM in vanilla game and the fan-made level, I could not easily discern which (if any) function controls this.

Any pointers on how to do this?

**Comment by dbrown1986** [comment]
I found it, finally. Thanks anyone who looked at this topic.

#start_staff[1].Doctor.Shrink.Consultant.Skill 1 1 1 100

In my case, wanting an ultimate doc, I think it would be:

#start_staff[1].Doctor.Shrink.Surgeon.Researcher.Consultant.Skill 1 1 1 1 1 100

I don't suppose we can manipulate their starting attention to detail?

**Reply by lewri** [reply]
Attention to detail, don't think so. I can't even remember where it's used in the game... Except maybe training?

Generally speaking, a staff member at 100 is going to get things pretty right anyway.. Though we did find that 3 x 100 skill all specialty doctors was a bit too powerful for the level.
